ci/test/lint-main/checks/check-python-discouraged.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# check-python-discouraged.sh — check discouraged commands.
+
+set -euo pipefail
+
+cd "$(dirname "$0")/../../../.."
+
+. misc/shlib/shlib.bash
+
+if [[ ! "${MZDEV_NO_PYTHON:-}" ]]; then
+
+  SINGLE_LINE_MATCHES=$(awk '/\.run\("testdrive",/ {print FILENAME ":" FNR} {prev_line = $0}' misc/python/materialize/**/*.py test/**/*.py)
+  MULTI_LINE_MATCHES=$(awk '/"testdrive",/ && prev_line ~ /\.run\($/ {print FILENAME ":" FNR} {prev_line = $0}' misc/python/materialize/**/*.py test/**/*.py)
+  # TODO
+  # MULTI_LINE_MATCHES=$(echo "$MULTI_LINE_MATCHES" | grep -v "misc/python/materialize/mzcompose/composition.py")
```

Comment:
Any ideas why this is not working in the script? It is working when executed directly in the terminal.

Reply:
Because it is in SINGLE_LINE_MATCHES, not MULTI_LINE_MATCHES

Reply:
Sorry, I was not properly explaining the problem. I believe that `MULTI_LINE_MATCHES` as such is correct.
The problem is that this line cannot be executed within the script, unlike when I run it directly in the terminal. That is, if I add `echo "got here"` in the following line, that line won't be executed / be reached at all. No error message will be printed but the return code will be 1.